src/mlpack/core/util/timers.hpp

```diff
@@ -14,23 +14,24 @@
 #ifndef MLPACK_CORE_UTILITIES_TIMERS_HPP
 #define MLPACK_CORE_UTILITIES_TIMERS_HPP
 
+#include <atomic>
+#include <chrono> // chrono library for cross platform timer calculation.
+#include <iomanip>
+#include <list>
 #include <map>
+#include <mutex>
 #include <string>
-#include <chrono> // chrono library for cross platform timer calculation.
 #include <thread> // std::thread is used for thread safety.
-#include <mutex>
-#include <list>
-#include <atomic>
 
 #if defined(_WIN32)
-  // uint64_t isn't defined on every windows.
-  #if !defined(HAVE_UINT64_T)
-    #if SIZEOF_UNSIGNED_LONG == 8
-      typedef unsigned long uint64_t;
-    #else
-      typedef unsigned long long  uint64_t;
-    #endif  // SIZEOF_UNSIGNED_LONG
-  #endif  // HAVE_UINT64_T
+// uint64_t isn't defined on every windows.
+#if !defined(HAVE_UINT64_T)
+#if SIZEOF_UNSIGNED_LONG == 8
+typedef unsigned long uint64_t;
+#else
+typedef unsigned long long uint64_t;
+#endif // SIZEOF_UNSIGNED_LONG
+#endif // HAVE_UINT64_T
```

Comment:
I think it was a little nicer to have the tabbing here before. :+1:

Reply:
Hmm, I am not sure if I modified this file, I will take a look.

Reply:
Yes, I need to blame kakoune for that